service/src/main/java/bio/terra/workspace/service/resource/ValidationUtils.java

```diff
@@ -38,21 +39,26 @@
   public static final Pattern AI_NOTEBOOK_INSTANCE_NAME_VALIDATION_PATTERN =
       Pattern.compile("(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)");
 
-  /** Resource names must be 1-1024 characters, using letters, numbers, dashes, and underscores and must not start with a dash or underscore.*/
+  /**
+   * Resource names must be 1-1024 characters, using letters, numbers, dashes, and underscores and
+   * must not start with a dash or underscore.
+   */
   public static final Pattern RESOURCE_NAME_VALIDATION_PATTERN =
       Pattern.compile("^[a-zA-Z0-9][-_a-zA-Z0-9]{0,1023}$");
 
   private static final String GOOG_PREFIX = "goog";
   private static final ImmutableList<String> GOOGLE_NAMES = ImmutableList.of("google", "g00gle");
+  private static final int MAX_RESOURCE_DESCRIPTION_NAME = 2048;
```

Comment:
My intuition says to make this bigger, but I don't have a solid technical reason.
Maybe we leave it as is and if anyone runs ever runs in to it with a legit purpose, we can bump it up.